docs/guide/dev/features/easyimage/README.md

```diff
@@ -0,0 +1,46 @@
+---
+category: inserting-images
+order: 40
+url: guide/dev_easyimage
+menu-title: Easy Image
+meta-title-short: Easy Image
+---
+<!--
+Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# Easy Image Plugin
+
+<info-box info=""> This feature was introduced in <strong>CKEditor 4.9</strong>. It is provided through an optional plugin that is not included in the CKEditor presets available from the [Download](https://ckeditor.com/ckeditor-4/download/) site and {@link guide/dev/widget_installation/README needs to be added to your custom build} with [CKBuilder](https://ckeditor.com/cke4/builder).</info-box>
+
+The optional [Easy Image](https://ckeditor.com/cke4/addon/easyimage) plugin introduces a new type of a captioned image widget that has the following capabilities:
+
+*   It allows for **uploading images** to [Cloud Services](https://ckeditor.com/ckeditor-cloud-services/) with customizable upload progress indication.
+*   It allows for adding **image captions** (that will not be separated from the image when its location changes).
```

Comment:
"when its location changes" that doesn't sound good in this context, something like "as image is moved" or sth? Ask Anna if you have doubts.

Reply:
Actually this phrase is copied from our "Enhanced Image" guide.

Reply:
Still that does not sound too good - but that's only me. @AnnaTomanek what do you think? I feel that "location" matches more to things that can have "absolute" address, like an estate or URL. Image content in the document seems to be more of a relative thing, it's always "next to xyz section" etc.

Reply:
Changed to "position" in both guides, makes sense.